docs/installation/using-with-jest.md

```diff
@@ -31,7 +31,38 @@ Then, you need to tell Jest to transform `.vue` files using `vue-jest`. You can
 }
 ```
 
-> **Note:** If you are using Babel 7 or higher, you will need to add [babel-bridge](https://github.com/babel/babel-bridge) to your devDependencies (`$ npm install --save-dev babel-core@^7.0.0-bridge.0`).
+### Using with Babel
+
+If you are goind to use `babel` and import vue single file components with `.vue` extension inside your tests, you will need to install babel and transform `.js` files with `babel-jest` .
```

Comment:
If you are going (going typo)

Reply:
Yes, excuse me, missed it, i should auto check for typos next time around.